stac_fastapi/elasticsearch/setup.py

```diff
@@ -28,7 +29,6 @@
         "pre-commit",
         "requests",
         "ciso8601",
-        "overrides",
```

Comment:
this should have been a core dep rather than an extra. We never ran it without also installing the dev deps, so it didn't surface.